src/resources/js/tickets-block.js

```diff
@@ -45,16 +45,48 @@ tribe.tickets.block = {
 		'.tribe-block__tickets__item__quantity__remove, .tribe-block__tickets__item__quantity__add',
 		function( e ) {
 			e.preventDefault();
-			var input  = $( this ).parent().find( 'input[type="number"]' );
-			var add    = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var input = $( this ).parent().find( 'input[type="number"]' );
+			var add = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var step = ( undefined !== Number( input[ 0 ].step ) ) ? Number( input[ 0 ].step ) : 1;
+			var originalValue = input[ 0 ].value;
 
-			// stepUp or stepDown the input according to the button that was clicked
-			add ? input[0].stepUp() : input[0].stepDown();
 
-			// Trigger the on Change for the input as it's not handled via stepUp() || stepDown()
-			input.trigger( 'change' );
+			// stepUp or stepDown the input according to the button that was clicked
+			// handle IE/Edge
+			if ( add ) {
+				var max = ( undefined !== Number( input[ 0 ].min ) ) ? Number( input[ 0 ].min ) : 0;
+
+				if ( typeof input[ 0 ].stepUp === 'function' ) {
```

Comment:
does this work in IE and Edge? because IE and Edge do not support `stepUp()` and `stepDown()`, if they are not functions, would the ticket quantities ever change in these browsers?

Reply:
Ah, but they do, the support is just buggy and throws an InvalidState error.

So yes, it works, but there should be an `else` that handles edge cases (no pun intended) where they aren't functions. So an addition is in order...